src/cascadia/WindowsTerminal/IslandWindow.cpp

```diff
@@ -326,6 +326,38 @@ void IslandWindow::OnSize(const UINT width, const UINT height)
         _windowCloseButtonClickedHandler();
         return 0;
     }
+    case WM_MOUSEWHEEL:
+    {
+        // This whole handler is a hack for GH#979.
+        //
+        // On some laptops, their trackpads won't scroll inactive windows
+        // _ever_. With our entire window just being one giant XAML Island, the
+        // touchpad driver thinks our entire window is inactive, and won't
+        // scroll the XAML island. On those types of laptops, we'll get a
+        // WM_MOUSEWHEEL here, in our root window, when the trackpad scrolls.
+        // We're going to take that message and manually plumb it through to our
+        // TermControl's, or anything else that implements IMouseWheelListener.
+
+        // https://msdn.microsoft.com/en-us/library/windows/desktop/ms645617(v=vs.85).aspx
+        // Important! Do not use the LOWORD or HIWORD macros to extract the x-
+        // and y- coordinates of the cursor position because these macros return
+        // incorrect results on systems with multiple monitors. Systems with
+        // multiple monitors can have negative x- and y- coordinates, and LOWORD
+        // and HIWORD treat the coordinates as unsigned quantities.
+        const til::point eventPoint{ GET_X_LPARAM(lparam), GET_Y_LPARAM(lparam) };
+        // This mouse event is relative to the display origin, not the window. Convert here.
+        const til::rectangle windowRect{ GetWindowRect() };
+        const auto origin = windowRect.origin();
+        const auto relative = eventPoint - origin;

```

Comment:
This can throw. Does the outer message loop catch exceptions or will this bubble out the top?